config/locales/forms/register_in_northern_ireland_forms/en.yml

```diff
@@ -2,13 +2,15 @@ en:
   waste_carriers_engine:
     register_in_northern_ireland_forms:
       new:
-        title: You can register in Northern Ireland
-        heading: You can register in Northern Ireland
-        paragraph_1: You can continue to renew in England, but you are encouraged to register with the appropriate authority for your region. This allows customers to verify that you’re a registered waste carrier.
-        paragraph_2: In Northern Ireland, you need to register with the Northern Ireland Environment Agency (NIEA). For details on how, visit
-        service_link: Register or renew as a waste carrier, broker or dealer (Northern Ireland)
-        paragraph_3: If you have any questions about this, please contact our helpline on 03708 506 506.
-        next_button: Continue to register in England
+        title: Register where you carry, broker or deal waste
+        heading: Register where you carry, broker or deal waste
+        paragraph_1_1: You must
+        service_link: register with the Northern Ireland Environment Agency (NIEA)
+        paragraph_1_2: if you carry, broker or deal waste in Northern Ireland (NI).
+        paragraph_2: You must register with the Environment Agency (EA) if you also carry, broker or deal waste in England.
+        paragraph_3: Select continue to register with the Environment Agency in England.
+        next_button: Continue
+        contact_ea: Contact the EA on 03708 506 506 if you have questions or need help to register.
```

Comment:
I don't think we should be using EA abbreviations, this should be the Environment Agency